Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -4966,7 +4966,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 1250;
-				LastUpgradeCheck = 1310;
+				LastUpgradeCheck = 1410;
```

Comment:
Upgraded workspace check without applying any of recommended options:

<img width="590" alt="Screenshot 2022-12-08 at 11 18 25" src="https://user-images.githubusercontent.com/2358722/206423320-14bcdec2-4160-4caf-8532-75445987639a.png">

Rationale:
- we can't update deployment target to iOS 12, as we support iOS 11+
- setting `CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER=YES` enforces `<...>` (instead of `"..."`) imports in [`Datadog.h`](https://github.com/DataDog/dd-sdk-ios/blob/10a027af39572717623c261dd3b8ce0dd231641d/Datadog/TargetSupport/Datadog/Datadog.h#L17-L18), which doesn't compile.